src/main/java/seedu/storage/Storage.java

```diff
@@ -0,0 +1,104 @@
+package storage;
+
+import seedu.duke.DukeException;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Scanner;
+
+public class Storage {
+
+    private final File file;
+    private final SimpleDateFormat ft = new SimpleDateFormat("dd MMM yyy h.mma");
```

Comment:
```suggestion
    private final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("dd MMM yyy h.mma");
```
Consider changing naming variable to match naming convention for constants.